pkg/tekton/metapipeline/client.go

```diff
@@ -0,0 +1,24 @@
+package metapipeline
+
+import (
+	"github.com/jenkins-x/jx/pkg/kube"
+	"github.com/jenkins-x/jx/pkg/tekton"
+)
+
+// Client defines the interface for meta pipeline creation and application.
+type Client interface {
```

Comment:
@ccojocar, this would be my take on a meta pipeline client. You see it in action [here](https://github.com/jenkins-x/jx/pull/5051/files#diff-cfadf679d93b353852ef9097113f3e53R404)